go/systests/account_reset_test.go

```diff
@@ -14,9 +14,10 @@ func assertResetBadgeState(t *testing.T, user *userPlusDevice, expectedDaysLeft
 	user.kickAutoresetd()
 	pollForTrue(t, g, func(i int) bool {
 		badges := getBadgeState(t, user)
-		g.Log.Debug("Iter loop %d badge state: %+v", i, badges)
+		g.Log.Warning("Iter loop %d badge state: %+v", i, badges)
 		if expectedDaysLeft > 0 {
 			daysLeft := int(time.Until(badges.ResetState.EndTime.Time()) / (time.Hour * 24))
+			g.Log.Warning("Iter loop %d cmp: %v vs %v", i, daysLeft, expectedDaysLeft)
```

Comment:
it should be better to use `t.Logf` in tests and leave `g.Log` to things that comes from non-test code